packages/types/src/types/provider/ovirt/Provider.ts

```diff
@@ -1,8 +1,9 @@
 import { V1beta1Provider } from '../../../models';
-import { Resource } from '../openshift/Resource';
+
+import { OVirtResource } from './Resource';
 
 // https://github.com/kubev2v/forklift/tree/main/pkg/controller/provider/web/ovirt/provider.go
-export interface Provider extends Resource {
+export interface OVirtProvider extends OVirtResource {
```

Comment:
The `Provider` interface is extending the OpenShift resource not the Ovirt version.
See the backend struct:  https://github.com/kubev2v/forklift/blob/f06a2436ff709186354c8a4b3deba6e1ccd8d779/pkg/controller/provider/web/ovirt/provider.go#LL177C2-L177C2
Applies to other providers.